### PR TITLE
Add label entity_key for the team entity type

### LINKED ISSUE #444: Impossible to create a Node entity_referencing a Team in Behat
## Description
`\Drupal\Driver\Fields\Drupal8\EntityReferenceHandler::expand()` can't retrieve the `label` entity_key for the Team entity type, therefore a `\Behat\Testwork\Call\Exception\FatalThrowableError` is thrown when one tries to create a Node (or whatever entity type) that references a Team entity.

## Steps to Reproduce
1. Create a node type with an entity_reference field to a Team.
2. Try creating a node that references an existing team using `\Drupal\Driver\DrupalDriver::createNode()`.

## Actual Behavior
A `\Behat\Testwork\Call\Exception\FatalThrowableError` is thrown with the message `Type error: Too few arguments to function Drupal\apigee_edge\Entity\AttributesAwareFieldableEdgeEntityBase::get(), 0 passed and exactly 1 expected`.

## Expected Behavior
The node gets created without errors.

## Screenshots
(none)

## Notes
(none)

## Version Info
```
drupal/apigee_edge                                        1.11.0           Apigee Edge for Drupal.
drupal/core                                               8.8.4            Drupal is an open source content management platform powering millions of websites and applications.
```

### PR DESCRIPTION
**Pull request by boobaa**
fixes #444

### CODE REVIEW

File: modules/apigee_edge_teams/src/Entity/Team.php [hunk 1]
```diff
@@ -65,6 +65,7 @@
  *   },
  *   entity_keys = {
  *     "id" = "name",
+ *     "label" = "displayName",
```
Comment (mxr576) [inline modules/apigee_edge_teams/src/Entity/Team.php:68]: What about other entity types? Is this information also missing from those?

**Review by arlina-espinoza — APPROVED** [review]
LGTM :+1:

**Comment by arlina-espinoza** [comment]
@googlebot I consent.

File: modules/apigee_edge_teams/src/Entity/Team.php (hunk 1 above)
Comment (mxr576) [reply modules/apigee_edge_teams/src/Entity/Team.php:68]: @arlina-espinoza the PR has been merged, does it mean that other entity types do not need this change?

Comment (mxr576) [reply modules/apigee_edge_teams/src/Entity/Team.php:68]: Also, if entity keys are changing, should not we add an update hook that updates the active entity definition on existing sites? 🤔

Comment (mxr576) [reply modules/apigee_edge_teams/src/Entity/Team.php:68]: @arlina-espinoza ?

**Pull request merged**